flink/pom.xml

```diff
@@ -97,44 +95,44 @@
 
     <dependency>
       <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-actor_${flink.scala.binary.version}</artifactId>
+      <artifactId>akka-actor_${scala.binary.version}</artifactId>
```

Comment:
What would be the case for having interpreter specific scala versions ? Now it seems that everything work with latest scala 2.10. I am more towards simplifying the build now, and making it more complex when actually needed.  Do we have a concrete case where this is needed now ?

 If we are talking here about actually scala 2.10 versus 2.11, I plan to handle that by profiles/modules.